defaults/main.yml

```diff
@@ -287,6 +299,21 @@ docker__pki_crt: 'default.crt'
 #
 # Name of the private key file used by Docker.
 docker__pki_key: 'default.key'
+
+                                                                   # ]]]
+# .. envvar:: docker__private_registry_keys [[[
+#
+# Dictionary with private registry name and keys for
+# secure client-server communication.
+#
+# .. code-block:: yaml
+#    :linenos:
+#
+#    docker__private_registry_keys:
+#      'myregistry1.domain.com': '/etc/pki/realm/domain/CA.crt'
+#      'myregistry2domain.com:5000': '/etc/pki/realm/domain/CA.crt'
+docker__private_registry_keys: {}
```

Comment:
you are right. That was my first thought. But as the certificate can be added through the debops.pki role (as described in the docs), this is not needed anymore. Thought I removed it already.